src/styles/blocks/contact.css

```diff
@@ -0,0 +1,111 @@
+.contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.contact__form {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.contact__form--text {
+  height: 48px;
+  border-radius: 8px;
+  background-color: #F8F8FA;
+  border: none;
+  color: #7E7E83;
+  font-size: 14px;
+  line-height: 19.6px;
+  box-sizing: border-box;
+  padding: 14px 16px;
+}
+
+.contact__form--text:hover {
+  border: 1px solid #D5D7DE;
+  color: #292929;
+}
+
+.contact__form--text:target {
+  border: 1px solid #292929;
+  color: #292929;
+}
+
+.contact__form--button {
+  height: 56px;
+  border-radius: 8px;
+  border: 2px solid #F0F1F4;
+  background-color: #F8F8FA;
+  font-weight: 700;
+  line-height: 27px;
+}
```

Comment:
text and button are not modifiers but other elements of the contact block! You could use `contact__text` and `contact__button`